### ISSUE TITLE
crash if you try and move a locked prim

### ISSUE THREAD

**Issue by nxkb** [AnimalLogic/AL_USDMaya#63]
While testing some of the various prim attributes defined in Metadata.h that can customize created maya nodes, I was a little too harshly punished for trying to move a locked prim.

## Reproduction steps
- Load a proxy shape with stage that has a prim defined like: 
```
    def Xform "locked" (
        al_usdmaya_lock = "transform"
    )
    {
        def Sphere "ball"
        {
        }
    }
```
- select it
- try to move it
- crashes with:
```
FATAL ERROR: Used null prim
in Usd_IssueFatalPrimAccessError at line 224 of /mnt/Samsung/luma/rez-build/usd/0.8.2.luma03/src/pxr/usd/lib/usd/primData.cpp
```

**Comment by dbaz** [comment]
Hi Nick,
I have simple fix locally to stop the crash and need to investigate exactly why the reason. I will have the fix up ASAP.

If you havn't done the fix already on your end, it's a return early in ProxyShape::lockTransformAttribute after the `GetPrimAtPath` if the returned prim is null. 

Also, I'm going to log one more issue publicly about the this lock feature that you may run into shortly.

- Baz

**Comment by dbaz** [comment]
I have an internal fix in for code review. There is also documentation for the Lock feature coming

**Comment by nxkb** [comment]
Awesome. Thanks for looking into it!

**Comment by dbaz** [comment]
Fix is in version AL_USDMaya-0.26
